crates/sncast/tests/e2e/account/add.rs

```diff
@@ -3,14 +3,13 @@ use crate::helpers::runner::runner;
 use camino::Utf8PathBuf;
 use indoc::indoc;
 use serde_json::json;
-use snapbox::cmd::{cargo_bin, Command};
 use std::fs;
-use tempfile::TempDir;
+use tempfile::tempdir;
 
 #[tokio::test]
 pub async fn test_happy_case() {
-    let accounts_file = "./tmp-a1/accounts.json";
-    _ = fs::remove_file(accounts_file);
+    let tempdir = tempdir().expect("Unable to create a temporary directory");
+    let accounts_file = "./accounts.json";
```

Comment:
nit: later on you join tempdir with accounts_file, yielding "/temp/path/to/./accounts.json", which is fine, but not very elegant.

Reply:
applicable to all the places you do this

Reply:
This is the way it was done before (e.g. in `test_happy_case_add_profile`), so I assumed it was done for readability purposes. Should I change this everywhere as a part of this PR?